app/src/pages/HomePage/HomePage.tsx

```diff
@@ -33,10 +33,10 @@ export const HomePage = (): React.ReactElement => {
   const { notdClient } = useGlobals();
   const [isEmailPopupShowing, setIsEmailPopopShowing] = React.useState(false);
   const [highestPricedTokenTransfer, setHighestPricedTokenTransfer] = React.useState<TokenTransfer| null>(null);
-  const [mostTradedToken, setMostTradedToken] = React.useState<TradedToken | null>(null);
+  const [mostTradedToken, setMostTradedToken] = React.useState<TradedToken | null >(null);
```

Comment:
formatting error here and below - please be sure to review the code yourself here on gituhb!